torchci/lib/bot/cancelWorkflowsOnCloseBot.ts

```diff
@@ -19,6 +23,18 @@ function cancelWorkflowsOnCloseBot(app: Probot): void {
       return;
     }
 
+    const diff = await ctx.octokit.rest.repos.compareCommits({
+      owner,
+      repo,
+      base: "main",
+      head: headSha,
+    });
+
+    if (diff.data.merge_base_commit.sha == headSha) {
+      // PR got made with main branch as head
```

Comment:
It's not a PR, but rather a workflow dispatch job...

Reply:
This bot only runs when a PR goes closed, so what happened is a PR got created with a head sha belonging to main, I think todays was because someone rebased poorly.  Then the PR got closed, the bot saw that and then canceled every job on that sha, which can also include workflow dispatch jobs